sdk/liftrqumulo/ci.yml

```diff
@@ -0,0 +1,36 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
```

Comment:
remove this file

Reply:
it is mentioned to created this file in point #11 in this doc - https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process. Please let me know, stil lthis file needs to be removed.